lib/parser/v1.ts

```diff
@@ -67,11 +67,11 @@ function checkForOldFormat(ruleSet: unknown) {
   // alpha format, and we'll throw
   if (isObject(ruleSet)) {
     for (const id in ruleSet) {
-      if (!Array.isArray(ruleSet[id])) {
+      if (ruleSet[id] && !Array.isArray(ruleSet[id])) {
```

Comment:
In #121 we added graceful handling of empty rules, so we must now check that `ruleSet[id]` is not undefined/null